persistence/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/multinode/AbstractClusteredPersistentEntitySpec.scala

```diff
@@ -149,44 +149,39 @@ abstract class AbstractClusteredPersistentEntitySpec(config: AbstractClusteredPe
 
     "send commands to target entity" in within(10.seconds) {
 
-      log.info(" 001 " + "*" * 30)
       val ref1 = registry.refFor[TestEntity]("1").withAskTimeout(remaining)
 
-      log.info(" 002 " + "*" * 30)
       // note that this is done on both node1 and node2
       val r1 = ref1.ask(TestEntity.Add("a"))
       r1.pipeTo(testActor)
       expectMsg(TestEntity.Appended("A"))
       enterBarrier("appended-A")
 
-      log.info(" 003 " + "*" * 30)
       val ref2 = registry.refFor[TestEntity]("2")
       val r2 = ref2.ask(TestEntity.Add("b"))
       r2.pipeTo(testActor)
       expectMsg(TestEntity.Appended("B"))
       enterBarrier("appended-B")
 
-      log.info(" 004 " + "*" * 30)
       val r3: Future[TestEntity.Evt] = ref2.ask(TestEntity.Add("c"))
       r3.pipeTo(testActor)
       expectMsg(TestEntity.Appended("C"))
       enterBarrier("appended-C")
 
-      log.info(" 005 " + "*" * 30)
       val r4: Future[TestEntity.State] = ref1.ask(TestEntity.Get)
       r4.pipeTo(testActor)
       expectMsgType[TestEntity.State].elements should ===(List("A", "A", "A"))
 
-      log.info(" 006 " + "*" * 30)
       val r5 = ref2.ask(TestEntity.Get)
       r5.pipeTo(testActor)
       expectMsgType[TestEntity.State].elements should ===(List("B", "B", "B", "C", "C", "C"))
 
-      expectAppendCount("1", 3)
-      expectAppendCount("2", 6)
-
-      log.info(" 007 " + "*" * 30)
-      enterBarrier("after-1")
+      try {
+        expectAppendCount("1", 3)
+        expectAppendCount("2", 6)
+      } finally {
+        enterBarrier("after-1")
+      }
```

Comment:
This `try` + `finally` is critical in preventing a cascading failure of the tests. 

This first test uses `expectAppendCount` which is only used on `node1`. When the `expectAppendCount` fails,  the `node1` execution of the test completes without ever reaching `enterBarrier("after-1")`.

It is not a bullet-proof solution (I've seen some runs on travis where all tests failed), but the following snippet is a failing run on travis where failure was contained:

```

[JVM-2] 2017-09-08 16:03:18,984 WARN  akka.serialization.Serialization(akka://AbstractClusteredPersistentEntitySpec) - Using the default Java serializer for class [com.lightbend.lagom.internal.persistence.cluster.ClusterStartupTaskActor$Execute$] which is not recommended because of performance implications. Use another serializer or disable this warning using the setting 'akka.actor.warn-about-java-serializer-usage'
[JVM-3] 2017-09-08 16:03:19,029 WARN  akka.serialization.Serialization(akka://AbstractClusteredPersistentEntitySpec) - Using the default Java serializer for class [com.lightbend.lagom.internal.persistence.cluster.ClusterStartupTaskActor$Execute$] which is not recommended because of performance implications. Use another serializer or disable this warning using the setting 'akka.actor.warn-about-java-serializer-usage'
[JVM-1] 2017-09-08 16:03:19,040 INFO  akka.cluster.Cluster(akka://AbstractClusteredPersistentEntitySpec) - Cluster Node [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:44009] - Leader is moving node [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:35529] to [Up]
[JVM-1] 2017-09-08 16:03:19,041 INFO  akka.cluster.Cluster(akka://AbstractClusteredPersistentEntitySpec) - Cluster Node [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:44009] - Leader is moving node [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423] to [Up]
[JVM-3] 2017-09-08 16:03:19,247 INFO  akka.cluster.singleton.ClusterSingletonManager - ClusterSingletonManager state change [Start -> Younger]
[JVM-3] 2017-09-08 16:03:19,248 INFO  akka.cluster.singleton.ClusterSingletonManager - ClusterSingletonManager state change [Start -> Younger]
[JVM-3] 2017-09-08 16:03:19,249 INFO  akka.cluster.singleton.ClusterSingletonManager - ClusterSingletonManager state change [Start -> Younger]
[JVM-3] 2017-09-08 16:03:19,249 INFO  akka.cluster.singleton.ClusterSingletonManager - ClusterSingletonManager state change [Start -> Younger]
[JVM-2] 2017-09-08 16:03:19,493 INFO  akka.cluster.singleton.ClusterSingletonManager - ClusterSingletonManager state change [Start -> Younger]
[JVM-2] 2017-09-08 16:03:19,493 INFO  akka.cluster.singleton.ClusterSingletonManager - ClusterSingletonManager state change [Start -> Younger]
[JVM-3] A PersistentEntity in a Cluster
[JVM-1] A PersistentEntity in a Cluster
[JVM-2] A PersistentEntity in a Cluster
[JVM-3] 2017-09-08 16:03:21,837 ERROR akka.actor.OneForOneStrategy - Ask timed out on [Actor[akka://AbstractClusteredPersistentEntitySpec/user/$b#1433629094]] after [100 ms]. Sender[null] sent message of type "akka.persistence.jdbc.query.JournalSequenceActor$GetMaxOrderingId$".
[JVM-3] akka.pattern.AskTimeoutException: Ask timed out on [Actor[akka://AbstractClusteredPersistentEntitySpec/user/$b#1433629094]] after [100 ms]. Sender[null] sent message of type "akka.persistence.jdbc.query.JournalSequenceActor$GetMaxOrderingId$".
[JVM-3] 	at akka.pattern.PromiseActorRef$$anonfun$1.apply$mcV$sp(AskSupport.scala:601)
[JVM-3] 	at akka.actor.Scheduler$$anon$4.run(Scheduler.scala:140)
[JVM-3] 	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:601)
[JVM-3] 	at scala.concurrent.BatchingExecutor$class.execute(BatchingExecutor.scala:109)
[JVM-3] 	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:599)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$TaskHolder.executeTask(LightArrayRevolverScheduler.scala:328)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$$anon$4.executeBucket$1(LightArrayRevolverScheduler.scala:279)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$$anon$4.nextTick(LightArrayRevolverScheduler.scala:283)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$$anon$4.run(LightArrayRevolverScheduler.scala:235)
[JVM-3] 	at java.lang.Thread.run(Thread.java:748)
[JVM-3] 2017-09-08 16:03:21,845 ERROR akka.actor.OneForOneStrategy - Ask timed out on [Actor[akka://AbstractClusteredPersistentEntitySpec/user/$b#1433629094]] after [100 ms]. Sender[null] sent message of type "akka.persistence.jdbc.query.JournalSequenceActor$GetMaxOrderingId$".
[JVM-3] akka.pattern.AskTimeoutException: Ask timed out on [Actor[akka://AbstractClusteredPersistentEntitySpec/user/$b#1433629094]] after [100 ms]. Sender[null] sent message of type "akka.persistence.jdbc.query.JournalSequenceActor$GetMaxOrderingId$".
[JVM-3] 	at akka.pattern.PromiseActorRef$$anonfun$1.apply$mcV$sp(AskSupport.scala:601)
[JVM-3] 	at akka.actor.Scheduler$$anon$4.run(Scheduler.scala:140)
[JVM-3] 	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:601)
[JVM-3] 	at scala.concurrent.BatchingExecutor$class.execute(BatchingExecutor.scala:109)
[JVM-3] 	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:599)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$TaskHolder.executeTask(LightArrayRevolverScheduler.scala:328)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$$anon$4.executeBucket$1(LightArrayRevolverScheduler.scala:279)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$$anon$4.nextTick(LightArrayRevolverScheduler.scala:283)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$$anon$4.run(LightArrayRevolverScheduler.scala:235)
[JVM-3] 	at java.lang.Thread.run(Thread.java:748)
[JVM-3] 2017-09-08 16:03:21,855 INFO  akka.remote.RemoteActorRefProvider$RemoteDeadLetterActorRef - Message [akka.persistence.jdbc.query.JournalSequenceActor$MaxOrderingId] from Actor[akka://AbstractClusteredPersistentEntitySpec/user/$b#1433629094] to Actor[akka://AbstractClusteredPersistentEntitySpec/deadLetters] was not delivered. [1] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-3] 2017-09-08 16:03:21,857 INFO  akka.remote.RemoteActorRefProvider$RemoteDeadLetterActorRef - Message [akka.persistence.jdbc.query.JournalSequenceActor$MaxOrderingId] from Actor[akka://AbstractClusteredPersistentEntitySpec/user/$b#1433629094] to Actor[akka://AbstractClusteredPersistentEntitySpec/deadLetters] was not delivered. [2] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-3] 2017-09-08 16:03:21,857 INFO  akka.remote.RemoteActorRefProvider$RemoteDeadLetterActorRef - Message [akka.persistence.jdbc.query.JournalSequenceActor$MaxOrderingId] from Actor[akka://AbstractClusteredPersistentEntitySpec/user/$b#1433629094] to Actor[akka://AbstractClusteredPersistentEntitySpec/deadLetters] was not delivered. [3] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-3] 2017-09-08 16:03:21,857 INFO  akka.remote.RemoteActorRefProvider$RemoteDeadLetterActorRef - Message [akka.persistence.jdbc.query.JournalSequenceActor$MaxOrderingId] from Actor[akka://AbstractClusteredPersistentEntitySpec/user/$b#1433629094] to Actor[akka://AbstractClusteredPersistentEntitySpec/deadLetters] was not delivered. [4] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-3] 2017-09-08 16:03:21,860 ERROR akka.actor.OneForOneStrategy - Ask timed out on [Actor[akka://AbstractClusteredPersistentEntitySpec/user/$b#1433629094]] after [100 ms]. Sender[null] sent message of type "akka.persistence.jdbc.query.JournalSequenceActor$GetMaxOrderingId$".
[JVM-3] akka.pattern.AskTimeoutException: Ask timed out on [Actor[akka://AbstractClusteredPersistentEntitySpec/user/$b#1433629094]] after [100 ms]. Sender[null] sent message of type "akka.persistence.jdbc.query.JournalSequenceActor$GetMaxOrderingId$".
[JVM-3] 	at akka.pattern.PromiseActorRef$$anonfun$1.apply$mcV$sp(AskSupport.scala:601)
[JVM-3] 	at akka.actor.Scheduler$$anon$4.run(Scheduler.scala:140)
[JVM-3] 	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:601)
[JVM-3] 	at scala.concurrent.BatchingExecutor$class.execute(BatchingExecutor.scala:109)
[JVM-3] 	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:599)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$TaskHolder.executeTask(LightArrayRevolverScheduler.scala:328)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$$anon$4.executeBucket$1(LightArrayRevolverScheduler.scala:279)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$$anon$4.nextTick(LightArrayRevolverScheduler.scala:283)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$$anon$4.run(LightArrayRevolverScheduler.scala:235)
[JVM-3] 	at java.lang.Thread.run(Thread.java:748)
[JVM-3] 2017-09-08 16:03:21,868 ERROR akka.actor.OneForOneStrategy - Ask timed out on [Actor[akka://AbstractClusteredPersistentEntitySpec/user/$b#1433629094]] after [100 ms]. Sender[null] sent message of type "akka.persistence.jdbc.query.JournalSequenceActor$GetMaxOrderingId$".
[JVM-3] akka.pattern.AskTimeoutException: Ask timed out on [Actor[akka://AbstractClusteredPersistentEntitySpec/user/$b#1433629094]] after [100 ms]. Sender[null] sent message of type "akka.persistence.jdbc.query.JournalSequenceActor$GetMaxOrderingId$".
[JVM-3] 	at akka.pattern.PromiseActorRef$$anonfun$1.apply$mcV$sp(AskSupport.scala:601)
[JVM-3] 	at akka.actor.Scheduler$$anon$4.run(Scheduler.scala:140)
[JVM-3] 	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:601)
[JVM-3] 	at scala.concurrent.BatchingExecutor$class.execute(BatchingExecutor.scala:109)
[JVM-3] 	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:599)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$TaskHolder.executeTask(LightArrayRevolverScheduler.scala:328)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$$anon$4.executeBucket$1(LightArrayRevolverScheduler.scala:279)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$$anon$4.nextTick(LightArrayRevolverScheduler.scala:283)
[JVM-3] 	at akka.actor.LightArrayRevolverScheduler$$anon$4.run(LightArrayRevolverScheduler.scala:235)
[JVM-3] 	at java.lang.Thread.run(Thread.java:748)
[JVM-2] - must send commands to target entity
[JVM-3] - must send commands to target entity *** FAILED ***
[JVM-3]   java.lang.AssertionError: assertion failed: block took 10.016 seconds, exceeding 10.000 seconds
[JVM-3]   at scala.Predef$.assert(Predef.scala:170)
[JVM-3]   at akka.testkit.TestKitBase$class.within(TestKit.scala:349)
[JVM-3]   at akka.testkit.TestKit.within(TestKit.scala:831)
[JVM-3]   at akka.testkit.TestKitBase$class.within(TestKit.scala:358)
[JVM-3]   at akka.testkit.TestKit.within(TestKit.scala:831)
[JVM-3]   at com.lightbend.lagom.scaladsl.persistence.multinode.AbstractClusteredPersistentEntitySpec$$anonfun$1$$anonfun$apply$mcV$sp$9.apply(AbstractClusteredPersistentEntitySpec.scala:150)
[JVM-3]   at org.scalatest.OutcomeOf$class.outcomeOf(OutcomeOf.scala:85)
[JVM-3]   at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[JVM-3]   at org.scalatest.Transformer.apply(Transformer.scala:22)
[JVM-3]   at org.scalatest.Transformer.apply(Transformer.scala:20)
[JVM-3]   ...
[JVM-1] - must send commands to target entity *** FAILED ***
[JVM-1]   0 did not equal 3 (AbstractClusteredPersistentEntitySpec.scala:143)
[JVM-3] - must run entities on specific node roles
[JVM-2] - must run entities on specific node roles
[JVM-1] - must run entities on specific node roles
[JVM-2] 2017-09-08 16:03:30,258 INFO  akka.actor.EmptyLocalActorRef - Message [akka.cluster.sharding.ShardCoordinator$Internal$GracefulShutdownReq] from Actor[akka://AbstractClusteredPersistentEntitySpec/system/sharding/TestEntity#52839518] to Actor[akka://AbstractClusteredPersistentEntitySpec/system/sharding/TestEntityCoordinator/singleton/coordinator] was not delivered. [1] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-2] 2017-09-08 16:03:30,355 INFO  akka.cluster.Cluster(akka://AbstractClusteredPersistentEntitySpec) - Cluster Node [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423] - Marked address [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423] as [Leaving]
[JVM-3] 2017-09-08 16:03:32,161 INFO  akka.cluster.Cluster(akka://AbstractClusteredPersistentEntitySpec) - Cluster Node [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:35529] - Leader is moving node [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423] to [Exiting]
[JVM-2] 2017-09-08 16:03:33,150 INFO  akka.cluster.Cluster(akka://AbstractClusteredPersistentEntitySpec) - Cluster Node [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423] - Exiting, starting coordinated shutdown
[JVM-2] 2017-09-08 16:03:33,154 INFO  akka.remote.RemoteActorRefProvider$RemoteDeadLetterActorRef - Message [akka.cluster.sharding.ShardRegion$GracefulShutdown$] without sender to Actor[akka://AbstractClusteredPersistentEntitySpec/deadLetters] was not delivered. [2] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-2] 2017-09-08 16:03:33,167 INFO  akka.cluster.singleton.ClusterSingletonManager - Exited [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423]
[JVM-2] 2017-09-08 16:03:33,178 INFO  akka.cluster.singleton.ClusterSingletonManager - Exited [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423]
[JVM-2] 2017-09-08 16:03:33,183 INFO  akka.cluster.Cluster(akka://AbstractClusteredPersistentEntitySpec) - Cluster Node [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423] - Exiting completed
[JVM-2] 2017-09-08 16:03:33,183 INFO  akka.cluster.Cluster(akka://AbstractClusteredPersistentEntitySpec) - Cluster Node [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423] - Shutting down...
[JVM-1] 2017-09-08 16:03:33,191 INFO  akka.cluster.Cluster(akka://AbstractClusteredPersistentEntitySpec) - Cluster Node [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:44009] - Exiting confirmed [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423]
[JVM-3] 2017-09-08 16:03:33,192 INFO  akka.cluster.Cluster(akka://AbstractClusteredPersistentEntitySpec) - Cluster Node [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:35529] - Exiting confirmed [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423]
[JVM-2] 2017-09-08 16:03:33,203 INFO  akka.cluster.Cluster(akka://AbstractClusteredPersistentEntitySpec) - Cluster Node [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423] - Successfully shut down
[JVM-2] 2017-09-08 16:03:33,224 INFO  akka.cluster.singleton.ClusterSingletonManager - Self removed, stopping ClusterSingletonManager
[JVM-2] 2017-09-08 16:03:33,238 INFO  akka.actor.LocalActorRef - Message [akka.cluster.GossipEnvelope] from Actor[akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:44009/system/cluster/core/daemon#-1194850138] to Actor[akka://AbstractClusteredPersistentEntitySpec/system/cluster/core/daemon#541389957] was not delivered. [3] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-2] 2017-09-08 16:03:33,238 INFO  akka.cluster.singleton.ClusterSingletonManager - Self removed, stopping ClusterSingletonManager
[JVM-2] 2017-09-08 16:03:33,239 INFO  akka.actor.RepointableActorRef - Message [akka.cluster.singleton.ClusterSingletonProxy$TryToIdentifySingleton$] from Actor[akka://AbstractClusteredPersistentEntitySpec/user/jdbcCreateTables-singletonProxy#-1025594874] to Actor[akka://AbstractClusteredPersistentEntitySpec/user/jdbcCreateTables-singletonProxy#-1025594874] was not delivered. [4] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-2] 2017-09-08 16:03:33,240 INFO  akka.actor.RepointableActorRef - Message [akka.cluster.singleton.ClusterSingletonManager$Internal$DelayedMemberRemoved] from Actor[akka://AbstractClusteredPersistentEntitySpec/user/jdbcCreateTables-singleton#268117013] to Actor[akka://AbstractClusteredPersistentEntitySpec/user/jdbcCreateTables-singleton#268117013] was not delivered. [5] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-2] 2017-09-08 16:03:33,242 INFO  akka.actor.RepointableActorRef - Message [akka.cluster.singleton.ClusterSingletonManager$Internal$DelayedMemberRemoved] from Actor[akka://AbstractClusteredPersistentEntitySpec/user/jdbcCreateTables-singleton#268117013] to Actor[akka://AbstractClusteredPersistentEntitySpec/user/jdbcCreateTables-singleton#268117013] was not delivered. [6] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-2] 2017-09-08 16:03:33,242 INFO  akka.actor.RepointableActorRef - Message [akka.cluster.singleton.ClusterSingletonManager$Internal$OldestChangedBuffer$OldestChanged] from Actor[akka://AbstractClusteredPersistentEntitySpec/user/jdbcCreateTables-singleton/$a#-1923656567] to Actor[akka://AbstractClusteredPersistentEntitySpec/user/jdbcCreateTables-singleton#268117013] was not delivered. [7] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-2] 2017-09-08 16:03:33,242 INFO  akka.actor.LocalActorRef - Message [akka.cluster.singleton.ClusterSingletonManager$Internal$DelayedMemberRemoved] from Actor[akka://AbstractClusteredPersistentEntitySpec/system/sharding/TestEntityCoordinator#444289960] to Actor[akka://AbstractClusteredPersistentEntitySpec/system/sharding/TestEntityCoordinator#444289960] was not delivered. [8] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-2] 2017-09-08 16:03:33,242 INFO  akka.actor.LocalActorRef - Message [akka.cluster.singleton.ClusterSingletonManager$Internal$DelayedMemberRemoved] from Actor[akka://AbstractClusteredPersistentEntitySpec/system/sharding/TestEntityCoordinator#444289960] to Actor[akka://AbstractClusteredPersistentEntitySpec/system/sharding/TestEntityCoordinator#444289960] was not delivered. [9] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-2] 2017-09-08 16:03:33,242 INFO  akka.actor.LocalActorRef - Message [akka.cluster.singleton.ClusterSingletonManager$Internal$OldestChangedBuffer$OldestChanged] from Actor[akka://AbstractClusteredPersistentEntitySpec/system/sharding/TestEntityCoordinator/$a#-2122755966] to Actor[akka://AbstractClusteredPersistentEntitySpec/system/sharding/TestEntityCoordinator#444289960] was not delivered. [10] dead letters encountered, no more dead letters will be logged. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-3] - must have support for graceful leaving
[JVM-2] - must have support for graceful leaving
[JVM-1] - must have support for graceful leaving
[JVM-2] 2017-09-08 16:03:33,373 INFO  akka.remote.RemoteActorRefProvider$RemotingTerminator - Shutting down remote daemon.
[JVM-2] 2017-09-08 16:03:33,373 INFO  akka.remote.RemoteActorRefProvider$RemotingTerminator - Remote daemon shut down; proceeding with flushing remote transports.
[JVM-1] 2017-09-08 16:03:33,424 ERROR akka.remote.EndpointWriter - AssociationError [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:44009] <- [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423]: Error [Shut down address: akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423] [
[JVM-1] akka.remote.ShutDownAssociation: Shut down address: akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423
[JVM-1] Caused by: akka.remote.transport.Transport$InvalidAssociationException: The remote system terminated the association because it is shutting down.
[JVM-1] ]
[JVM-3] 2017-09-08 16:03:33,448 INFO  akka.remote.RemoteActorRefProvider$RemotingTerminator - Shutting down remote daemon.
[JVM-3] 2017-09-08 16:03:33,449 INFO  akka.remote.RemoteActorRefProvider$RemotingTerminator - Remote daemon shut down; proceeding with flushing remote transports.
[JVM-2] 2017-09-08 16:03:33,481 INFO  akka.remote.Remoting - Remoting shut down
[JVM-2] 2017-09-08 16:03:33,482 INFO  akka.remote.RemoteActorRefProvider$RemotingTerminator - Remoting shut down.
[JVM-3] 2017-09-08 16:03:33,486 INFO  akka.actor.LocalActorRef - Message [akka.remote.transport.AssociationHandle$InboundPayload] without sender to Actor[akka://AbstractClusteredPersistentEntitySpec/system/endpointManager/reliableEndpointWriter-akka.tcp%3A%2F%2FAbstractClusteredPersistentEntitySpec%40localhost%3A44009-0/endpointWriter/endpointReader-akka.tcp%3A%2F%2FAbstractClusteredPersistentEntitySpec%40localhost%3A44009-0#310232175] was not delivered. [5] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-1] 2017-09-08 16:03:33,494 ERROR akka.remote.EndpointWriter - AssociationError [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:44009] <- [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:35529]: Error [Shut down address: akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:35529] [
[JVM-1] akka.remote.ShutDownAssociation: Shut down address: akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:35529
[JVM-1] Caused by: akka.remote.transport.Transport$InvalidAssociationException: The remote system terminated the association because it is shutting down.
[JVM-1] ]
[JVM-2] 2017-09-08 16:03:33,514 INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown initiated...
[JVM-1] 2017-09-08 16:03:33,516 INFO  akka.actor.LocalActorRef - Message [akka.remote.testconductor.Controller$ClientDisconnected$] without sender to Actor[akka://AbstractClusteredPersistentEntitySpec/user/controller/127.0.0.1:35978-server1#1827602037] was not delivered. [3] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-1] 2017-09-08 16:03:33,545 INFO  akka.remote.RemoteActorRefProvider$RemotingTerminator - Shutting down remote daemon.
[JVM-1] 2017-09-08 16:03:33,546 INFO  akka.remote.RemoteActorRefProvider$RemotingTerminator - Remote daemon shut down; proceeding with flushing remote transports.
[JVM-3] 2017-09-08 16:03:33,552 ERROR akka.remote.EndpointWriter - AssociationError [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:35529] <- [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423]: Error [Shut down address: akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423] [
[JVM-3] akka.remote.ShutDownAssociation: Shut down address: akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:45423
[JVM-3] Caused by: akka.remote.transport.Transport$InvalidAssociationException: The remote system terminated the association because it is shutting down.
[JVM-3] ]
[JVM-3] 2017-09-08 16:03:33,559 INFO  akka.actor.LocalActorRef - Message [akka.remote.EndpointWriter$FlushAndStop$] from Actor[akka://AbstractClusteredPersistentEntitySpec/system/endpointManager/reliableEndpointWriter-akka.tcp%3A%2F%2FAbstractClusteredPersistentEntitySpec%40localhost%3A45423-1#1801563512] to Actor[akka://AbstractClusteredPersistentEntitySpec/system/endpointManager/reliableEndpointWriter-akka.tcp%3A%2F%2FAbstractClusteredPersistentEntitySpec%40localhost%3A45423-1/endpointWriter#-1146516817] was not delivered. [6] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-3] 2017-09-08 16:03:33,559 INFO  akka.actor.LocalActorRef - Message [akka.remote.EndpointWriter$AckIdleCheckTimer$] from Actor[akka://AbstractClusteredPersistentEntitySpec/system/endpointManager/reliableEndpointWriter-akka.tcp%3A%2F%2FAbstractClusteredPersistentEntitySpec%40localhost%3A45423-1/endpointWriter#-1146516817] to Actor[akka://AbstractClusteredPersistentEntitySpec/system/endpointManager/reliableEndpointWriter-akka.tcp%3A%2F%2FAbstractClusteredPersistentEntitySpec%40localhost%3A45423-1/endpointWriter#-1146516817] was not delivered. [7] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[JVM-3] 2017-09-08 16:03:33,593 INFO  akka.remote.Remoting - Remoting shut down
[JVM-3] 2017-09-08 16:03:33,595 INFO  akka.remote.RemoteActorRefProvider$RemotingTerminator - Remoting shut down.
[JVM-3] 2017-09-08 16:03:33,599 INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown initiated...
[JVM-2] 2017-09-08 16:03:33,608 INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown completed.
[JVM-1] 2017-09-08 16:03:33,628 INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown initiated...
[JVM-1] 2017-09-08 16:03:33,631 INFO  akka.remote.Remoting - Remoting shut down
[JVM-3] [ERROR] [09/08/2017 16:03:33.635] [AbstractClusteredPersistentEntitySpec-akka.actor.default-dispatcher-3] [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:44009/system/sharding/SlickTestEntityReadSide%24TestEntityReadSideProcessorCoordinator/singleton/coordinator] swallowing exception during message send (akka.remote.RemoteTransportExceptionNoStackTrace: Attempted to send remote message but Remoting is not running.)
[JVM-3] [ERROR] [09/08/2017 16:03:33.650] [AbstractClusteredPersistentEntitySpec-akka.actor.default-dispatcher-4] [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:44009/system/remote-watcher] swallowing exception during message send (akka.remote.RemoteTransportExceptionNoStackTrace: Attempted to send remote message but Remoting is not running.)
[JVM-3] [ERROR] [09/08/2017 16:03:33.648] [AbstractClusteredPersistentEntitySpec-akka.actor.default-dispatcher-5] [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:44009/system/remote-watcher] swallowing exception during message send (akka.remote.RemoteTransportExceptionNoStackTrace: Attempted to send remote message but Remoting is not running.)
[JVM-3] [ERROR] [09/08/2017 16:03:33.656] [AbstractClusteredPersistentEntitySpec-akka.actor.default-dispatcher-3] [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:44009/system/sharding/TestEntity] swallowing exception during message send (akka.remote.RemoteTransportExceptionNoStackTrace: Attempted to send remote message but Remoting is not running.)
[JVM-3] [ERROR] [09/08/2017 16:03:33.657] [AbstractClusteredPersistentEntitySpec-akka.actor.default-dispatcher-3] [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:44009/system/sharding/TestEntityCoordinator/singleton/coordinator] swallowing exception during message send (akka.remote.RemoteTransportExceptionNoStackTrace: Attempted to send remote message but Remoting is not running.)
[JVM-2] 2017-09-08 16:03:33,665 INFO  application - Shutting down connection pool.
[JVM-1] [ERROR] [09/08/2017 16:03:33.629] [AbstractClusteredPersistentEntitySpec-akka.actor.default-dispatcher-3] [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:35529/system/remote-watcher] swallowing exception during message send (akka.remote.RemoteTransportExceptionNoStackTrace: Attempted to send remote message but Remoting is not running.)
[JVM-1] [ERROR] [09/08/2017 16:03:33.631] [AbstractClusteredPersistentEntitySpec-akka.actor.default-dispatcher-6] [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:35529/system/sharding/TestEntity] swallowing exception during message send (akka.remote.RemoteTransportExceptionNoStackTrace: Attempted to send remote message but Remoting is not running.)
[JVM-1] [ERROR] [09/08/2017 16:03:33.683] [AbstractClusteredPersistentEntitySpec-akka.actor.default-dispatcher-6] [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:35529/system/sharding/SlickTestEntityReadSide%24TestEntityReadSideProcessor] swallowing exception during message send (akka.remote.RemoteTransportExceptionNoStackTrace: Attempted to send remote message but Remoting is not running.)
[JVM-1] [ERROR] [09/08/2017 16:03:33.689] [AbstractClusteredPersistentEntitySpec-akka.actor.default-dispatcher-17] [akka.tcp://AbstractClusteredPersistentEntitySpec@localhost:35529/system/remote-watcher] swallowing exception during message send (akka.remote.RemoteTransportExceptionNoStackTrace: Attempted to send remote message but Remoting is not running.)
[JVM-2] Run completed in 26 seconds, 648 milliseconds.
[JVM-2] Total number of tests run: 3
[JVM-2] Suites: completed 1, aborted 0
[JVM-2] Tests: succeeded 3, failed 0, canceled 0, ignored 0, pending 0
[JVM-2] All tests passed.
[JVM-3] 2017-09-08 16:03:33,727 INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown completed.
[JVM-1] 2017-09-08 16:03:33,734 INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown completed.
[JVM-3] 2017-09-08 16:03:33,764 INFO  application - Shutting down connection pool.
[JVM-1] 2017-09-08 16:03:33,771 INFO  application - Shutting down connection pool.
[JVM-1] Run completed in 26 seconds, 679 milliseconds.
[JVM-1] Total number of tests run: 3
[JVM-1] Suites: completed 1, aborted 0
[JVM-1] Tests: succeeded 2, failed 1, canceled 0, ignored 0, pending 0
[JVM-1] *** 1 TEST FAILED ***
[JVM-3] Run completed in 26 seconds, 700 milliseconds.
[JVM-3] Total number of tests run: 3
[JVM-3] Suites: completed 1, aborted 0
[JVM-3] Tests: succeeded 2, failed 1, canceled 0, ignored 0, pending 0
[JVM-3] *** 1 TEST FAILED ***
[error] Failed: com.lightbend.lagom.scaladsl.persistence.slick.SlickClusteredPersistentEntitySpecMultiJvmNode1
[error] Failed: com.lightbend.lagom.scaladsl.persistence.slick.SlickClusteredPersistentEntitySpecMultiJvmNode3
```